sedai/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```

Comment:
If there are no metrics to list here, I believe this file can be deleted. @coignetp could you confirm?

Reply:
deleted